providers/gitlab/resources/gitlab.lr

```diff
@@ -121,11 +123,21 @@ gitlab.project.approvalSettings @defaults("approvalsBeforeMerge requirePasswordT
 
 
 // GitLab protected branch
-gitlab.project.repository.protectedBranch @defaults("name allowForcePush") {
+gitlab.project.protectedBranch @defaults("name allowForcePush") {
   // Branch name
   name string
   // Whether force push is allowed
   allowForcePush bool
   // Whether this is the default branch
   defaultBranch bool
+}
+
+// GitLab project file
+gitlab.project.file @defaults("path type") {
```

Comment:
I think this resource should be private since it can only be used via `gitlab.project.projectFiles`